milli/src/update/index_documents/transform.rs

```diff
@@ -677,6 +774,39 @@ impl<'a, 'i> Transform<'a, 'i> {
     }
 }
 
+/// Merge all the obks in the order we see them.
+fn merge_obkvs_and_operations<'a>(_key: &[u8], obkvs: &[Cow<'a, [u8]>]) -> Result<Cow<'a, [u8]>> {
```

Comment:
Huum, before making this change I would like to share with you the reason why it's here;

The way it ended up being here is because I wanted the `Operation` enum to stay specific to the `transform` module.
There are so many operations everywhere in milli that I thought it would be good to try to reduce the scope of this one.

But, since this function needs to take into account this enum, I was not able to keep it where it was before.

What do you think? Should we give up on the locality of the `Operation` for the sake of consistency? :thinking:

Reply:
I would prefer that we put back this merge function in the other module. As we will not expose this `Operation` type to the user, it is not that important 😊